Source/WebCore/platform/text/mac/TextBoundaries.mm

```diff
@@ -91,16 +91,24 @@ static CFStringTokenizerRef tokenizerForString(CFStringRef str)
 }
 
 // Simple case: A word is a stream of characters delimited by a special set of word-delimiting characters.
+template<bool is8Bit = false>
```

Comment:
Instead of boolean, can we define an enum class for this?
e.g. `CharType : bool { Is8Bit, Is16Bit };`

Reply:
I did that originally, since it makes the call site more understandable; but it felt a lot less readable within the function, where it is very intrusive. Also, since this file is all public and not namespaced, it injects CharType into the wider world; idk if we want that.

Reply:
ok, sounds reasonable to me.